src/main/docker/Dockerfile.agent

```diff
@@ -0,0 +1,20 @@
+####
+#
+# Create a base image for building web+java properties. Add support for nodeJS, NPM, and mvn
+#
+####
+FROM fabric8/java-alpine-openjdk11-jre
+
+# add mvn, npm, nodejs, hugo
+RUN apk update
```

Comment:
This is useless as we tell after to not use nor create any cache during the `add`command